setup.py

```diff
@@ -17,14 +22,14 @@ def get_version():
     """
     locals_dict = {}
     globals_dict = {}
-    with open(os.path.join('nilearn', 'version.py')) as fp:
+    with open(os.path.join(__curdir__, 'nilearn', 'version.py')) as fp:
         exec(fp.read(), globals_dict, locals_dict)
 
     return locals_dict['__version__']
 
 DISTNAME = 'nilearn'
 DESCRIPTION = 'Statistical learning for neuroimaging in Python'
-LONG_DESCRIPTION = open('README.rst').read()
+LONG_DESCRIPTION = open(os.path.join(__curdir__, 'README.rst')).read()
```

Comment:
BTW, instead of appending `__curdir__` everywhere, wouldn't it be easier to chdir at the beginning of the script?